src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -58,24 +59,24 @@ public sealed class Arguments : TransformInputBase
         internal const string ShortName = "Onnx";
         internal const string LoaderSignature = "OnnxTransform";
 
-        public readonly string Input;
-        public readonly string Output;
-        public readonly ColumnType OutputType;
+        public readonly string[] Inputs;
+        public readonly string[] Outputs;
+        public readonly ColumnType[] OutputTypes;
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "ONNXSCOR",
-                verWrittenCur: 0x00010001, // Initial
-                verReadableCur: 0x00010001,
+                verWrittenCur: 0x00010002, // Initial
+                verReadableCur: 0x00010002,
```

Comment:
>verReadableCur: 0x00010002, [](start = 16, length = 27)

would be nice to add comment what exactly changed, like // Inputs and Outputs columns now array of strings.

Reply:
done

---
In reply to: [233267446](https://github.com/dotnet/machinelearning/pull/1586#discussion_r233267446) [](ancestors = 233267446)